nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/java/org/nuxeo/ecm/platform/audit/service/LogEntryProvider.java

```diff
@@ -340,6 +343,14 @@ public LogEntry getLogEntryByID(long id) {
         return doPublish(query.getResultList());
     }
 
+    /**
+     * A string representation of an Operator
+     */
+    protected String toString(Operator operator) {
+        if (Operator.STARTSWITH.equals(operator)) return LIKE;
```

Comment:
Could you add the brackets and convert the if to an if/else block ? To confirm with @efge if it's the correct code style we want.